### PR TITLE
OADP-483: Warn about too frequent backup schedule

### PR DESCRIPTION
**Pull request by RichardHoch**
OADP 1.1.0, OCP 4.6+

Resolves https://issues.redhat.com/browse/OADP-483 by adding a short warning text to https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html-single/backup_and_restore/index#oadp-scheduling-backups_backing-up-applications.

Preview: http://file.emea.redhat.com/rhoch/backup_sched_warn/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.html#oadp-scheduling-backups_backing-up-applications

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

**Comment by mperetzred** [comment]
lgtm

**Comment by RichardHoch** [comment]
@sbeskin-redhat -- please review.

File: modules/oadp-scheduling-backups.adoc [hunk 1]
```diff
@@ -8,6 +8,13 @@
 
 You schedule backups by creating a `Schedule` custom resource (CR) instead of a `Backup` CR.
 
+[WARNING]
+====
+Leave enough time in your backup schedule for a backup to finish before another backup is created.
+
+For example, if a backup of a namespace typically takes 10 minutes, do not schedule backup more frequently than every 15 minutes.
+====
```
Comment (sbeskin-redhat) [inline modules/oadp-scheduling-backups.adoc:16]: @RichardHoch If you treat 'backup' as 'a single action of backing up', then I would say 'do not schedule backups' to be consistent.

**Comment by RichardHoch** [comment]
@sbeskin-redhat Fixed the problem; please review again.

**Comment by sbeskin-redhat** [comment]
@RichardHoch  Looks fine

**Review by bergerhoffer — COMMENTED** [review]
Merge review for OADP GA: LGTM

**Pull request merged**